tests/ami/scripts/test_rpm.sh.j2

```diff
@@ -57,6 +57,16 @@ id
 echo_with_date ""
 echo_with_date "==========================="
 
+# If it is a centos 8, then change repo source to centos vault.
```

Comment:
Please just add a comment on why we need to that - CentOS 8 now being EOL and repo not being active anymore.